standard-java/src/main/java/com/jetbrains/inspections/Java8Inspections.java

```diff
@@ -137,47 +116,32 @@ private void replaceMapForEachWithEntrySetLoop(Map<String, Integer> map, Map<Str
     private List<String> replaceWithCollectAndMap() {
         List<String> result = getStrings();
 
-        for (String line : stringArray) {
-            if (line != null) {
-                for (String word : line.split(" ")) {
-                    result.add(word.substring(0, 3));
-                }
-            }
-        }
+        Arrays.stream(stringArray)
+              .filter(Objects::nonNull)
+              .flatMap(line -> Arrays.stream(line.split(" ")))
+              .map(word -> word.substring(0, 3))
+              .forEach(result::add);
         return result;
     }
 
     private List<String> getListOfAllNonEmptyStringValues(Map<String, List<String>> map) {
         List<String> result = getStrings();
-        for (Map.Entry<String, List<String>> entry : map.entrySet()) {
-            if (entry.getKey()
-                    .isEmpty()) {
-                continue;
-            }
-            List<String> list = entry.getValue();
-            if (list == null) {
-                continue;
-            }
-            for (String str : list) {
-                String trimmed = str.trim();
-                if (trimmed.isEmpty()) {
-                    continue;
-                }
-                result.add(trimmed);
-            }
-        }
+        map.entrySet()
```

Comment:
This is quite a complex Stream operation, perhaps it should be split up for clarity?